singular.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * The template for displaying all pages, single posts and attachments
+ *
+ * This is a new template file that WordPress introduced in
+ * version 4.2. Note that it uses conditional logic to display
```

Comment:
Yes, you're right. Good point. The current back-compat.php enforces 4.2. Do you want to enforce 4.3+ only? If not, we'd have to do some fancy workarounds to load a separate single.php or page.php. Or include them as-is which would make singular.php moot.